README.md

```diff
@@ -1 +1,43 @@
-# graasp-template-backend
+# graasp-plugin-validation
+
+# Purpose
+The Graasp Explorer is designed to be a platform to share and explore open educational resources. Inappropriate content, such as dirty words, aggressive language, hate speech and etc., should be prohibited from publishing.
+
+# Introduction
+## Validation Process
+There will be different validation processes to check different aspects. Each validation process will have a record in DB.
+
+### Existing
+- Bad Words Detection (JS)
+### In progress
+- Aggressive and Hate Speech Classification (Python)
+- Image Checking
+
+## Status
+
+### Status for item-validation
+
+**Pending**: The process is in progress.
+
+**Success**: The item passes the test and can be published.
+
+**Failure**: The item is suspicious, and could not be published at the moment. A failed process will wait for manual review from administrator.
+
+### Status for item-validation-review
+
+**Pending**: The item-validation record is waiting for manual review.
+
+**Accepted**: The admin accepts the result of the validation, which means that the item is indeed problematic and the publication cannot proceed
+
+**Rejected**: Admin rejects the result of validation => The item is OK, and will be published.
+The admin rejects the result of the validation, which means that the item is OK and the publication cannot proceed
```

Comment:
Let's not mention about publication, or describe it as an example.
`The item is valid as if it passed the test. It can, for example, be published without issues.`